plugins/modules/packaging/os/portage.py

```diff
@@ -228,22 +240,37 @@
 
 import os
 import re
+import sys
+import traceback
 
-from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+from ansible.module_utils.common.respawn import has_respawned, respawn_module
 from ansible.module_utils.common.text.converters import to_native
 
 
+try:
+    from portage.dbapi import vartree
+    from portage.exception import InvalidAtom
+    HAS_PORTAGE = True
+    PORTAGE_IMPORT_ERROR = None
```

Comment:
It's best to move these to an else-block which specifically exists for such cases.